iree/compiler/Conversion/LinalgToSPIRV/LinalgTileAndFusePass.cpp

```diff
@@ -506,6 +527,47 @@ static void applyVectorTransformation(FuncOp funcOp) {
   }
 }
 
+//====---------------------------------------------------------------------===//
+// Patterns to tile convolution window dimensions
+//====---------------------------------------------------------------------===//
+
+/// Populates `patterns` with patterns that tiles a linalg.conv along filter's
+/// height, width, input channel dimensions with tile sizes 1, 1, 4.
+static void populateTilingConvFilterPatterns(MLIRContext *context,
```

Comment:
Not sure why the tile size function is different here. These should be set in the `LaunchConfig`. Trying to keep op-specific decisions out of this pass.

Reply:
Good point! Done.